### PR TITLE
[d3-chord] Add ribbonArrow()

### PR DESCRIPTION
**Pull request by simPod**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/d3-chord/index.d.ts [hunk 1]
```diff
@@ -473,3 +473,30 @@ export function ribbon<Datum, SubgroupDatum>(): RibbonGenerator<any, Datum, Subg
  * The third generic corresponds to the datum type of the chord subgroup, i.e. source or target of the cord. The default type is ChordSubgroup.
  */
 export function ribbon<This, Datum, SubgroupDatum>(): RibbonGenerator<This, Datum, SubgroupDatum>;
+
+/**
+ * Creates a new arrow ribbon generator with the default settings.
+ */
+export function ribbonArrow(): RibbonGenerator<any, Ribbon, RibbonSubgroup>;
```
Comment (Methuselah96) [inline types/d3-chord/index.d.ts:480]: Can  you make a `RibbonArrowGenerator` that has [the `headRadius` property](https://github.com/d3/d3-chord#ribbonArrow_headRadius) which `ribbonArrow()` would then return?

**Review by Methuselah96 — CHANGES_REQUESTED** [review]
Thanks! Sorry I missed this in the 2.0 release. Must have slipped through.

File: types/d3-chord/index.d.ts (hunk 1 above)
Comment (simPod) [reply types/d3-chord/index.d.ts:480]: Ah right, I thought so

Comment (simPod) [reply types/d3-chord/index.d.ts]: Ok I tried to add it. Not sure what to write in docs though. Should I copy paste it from https://github.com/d3/d3-chord?

**Review by Methuselah96 — APPROVED** [review]

**Comment by simPod** [comment]
Ready to merge

**Comment by simPod** [comment]
Ok this is superfast. If I knew that I would not bother creating local d.ts file 🙃

**Comment by Methuselah96** [comment]
Often it takes a while for people to review PRs. So it can take anywhere from hours to weeks.

**Pull request merged**